src/haddock/libs/libcli.py

```diff
@@ -40,6 +40,24 @@ def add_rundir_arg(ap):
         )
 
 
+def add_ncores_arg(ap):
+    """Add number of cores option."""
+    ap.add_argument(
+        "-n",
+        "--ncores",
+        dest='ncores',
+        help=(
+            "The number of threads to use. Uses 1 if not specified. "
```

Comment:
Ok - that answers my previous questions :-)  All good!

Reply:
Yes. `haddock3-clean -h` will tell also :wink: